include/model/CResourceMonitor.h

```diff
@@ -259,7 +259,7 @@ class MODEL_EXPORT CResourceMonitor {
     TMemoryUsageReporterFunc m_MemoryUsageReporter;
 
     //! Keep track of times of anomaly detector allocation failures
-    TTimeSizeMap m_AllocationFailures;
+    TSizeTimePr m_AllocationFailures;
```

Comment:
The comment is now wrong.

But I think the code would be much clearer overall if there were two separate member variables, namely `m_AllocationFailureCount` and `m_LastAllocationFailureTime`. Readers should be able to guess what those mean without even referring to the comment.

Reply:
This makes sense. Thank you.